calibration_date: '2025-03-17 17:49:11'
hard_iron_offset:
  x: -21.021464426469
  y: -1.7112762650692348
  z: 7.214548379527644
soft_iron_correction:
  x:
  - 104.46515900655173
  - 11.274760379930013
  - -2.640469045038998
  y:
  - 11.27476037993002
  - 69.53027466832846
  - 13.378717382379522
  z:
  - -2.6404690450390027
  - 13.378717382379515
  - 87.94653364288752
